src/main/java/org/apache/ibatis/migration/Change.java

```diff
@@ -37,6 +37,13 @@ public Change(BigDecimal id, String appliedTimestamp, String description) {
     this.description = description;
   }
 
+  public Change(BigDecimal id, String appliedTimestamp, String description, String filename) {
```

Comment:
Added this so I could have a single constructor that takes all params and not require a setter to add the the filename (I think that was the field missing)